### PR TITLE
[Keyboard] New Keyboard - Mokulua

### PR DESCRIPTION
**Pull request by kylemccreery**
## Description

Initial PR for adding the Mokulua keyboard kit by MechWild. This is the default, VIA, and Silly keymaps for Standard layout, and the default and VIA keymaps for Mirrored layout.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x]I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mechwild/mokulua/mirrored/config.h [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+#define LOCKING_RESYNC_ENABLE
+
+/* If defined, GRAVE_ESC will always act as ESC when CTRL is held.
+ * This is useful for the Windows task manager shortcut (ctrl+shift+esc).
+ */
+//#define GRAVE_ESC_CTRL_OVERRIDE
+
+/* disable these deprecated features by default */
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
```
Comment (drashna) [inline keyboards/mechwild/mokulua/mirrored/config.h:97]:
```suggestion
```

File: keyboards/mechwild/mokulua/standard/config.h [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+//#define NO_PRINT
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+
+/* disable these deprecated features by default */
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
```
Comment (drashna) [inline keyboards/mechwild/mokulua/standard/config.h:134]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by kylemccreery** [comment]
@drashna Are those not needed/suggested to include anymore going forward?

**Comment by drashna** [comment]
> @drashna Are those not needed/suggested to include anymore going forward?

Not entirely sure what you mean. 

But the action function and action macro features have been removed from QMK.  So the defines don't do anything anymore.

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**